src/front/app/routes/login.js

```diff
@@ -37,7 +37,19 @@ class LoginRoute extends React.Component {
                     success: true,
                 }));
                 setTimeout(() => {
-                    this.props.history.push('/');
+                    //console.log(this.props.history.location.pathname);
+                    if(this.props.history.location.pathname==='/')
+                    {
+                        this.props.history.go('/#/');
+                    }
+                    if(this.props.history.location.pathname==='/login')
+                    {
+                        this.props.history.push('/');
+                        window.location.reload();
```

Comment:
Why is a `.reload` here neccesary?

Reply:
the page is stuck without the reload

Reply:
I just tested this, and I see what you mean.